internal/setup/setup_test.go

```diff
@@ -1063,6 +1063,174 @@ var setupTests = []setupTest{{
 		`,
 	},
 	relerror: `invalid slice definition filename: "a.yaml"`,
+}, {
+	summary: "Specify generate: manifest",
```

Comment:
What is the second package in this test used for?

Reply:
At this point, I have no idea, lol. Removing it.

Reply:
:D, can you also quote the `generate: manifest` in the summary?